build/build.ts

```diff
@@ -285,24 +283,15 @@ async function generateFrontendJs(): Promise<void> {
 
   await bundle.write({
     preferConst: true,
-    format: "esm",
-    file: outputFile
-  });
-
-  const webpackConf = {
```

Comment:
Imho rollup does similar jobs comparing with webpack,  so that we can remove webpack for the build